src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Just widening policy used by getXConfigValue() JIT-EE call won't be sufficient for the following reasons:

consider the following config value from clrconfigvalues.h

RETAIL_CONFIG_DWORD_INFO_EX(INTERNAL_UseRyuJit, W("UseRyuJIT"), 0, "Set to 1 by .NET 4.6 installer to indicate RyuJIT should be used, not JIT64.", CLRConfig::IgnoreEnv | CLRConfig::IgnoreHKCU | CLRConfig::IgnoreConfigFiles)

Look at the config options: this specific setting cannot come from env, cannot be under HKCU nor from config files.

We cannot suddenly allow some config settings to come from sources which are not supported previously for a reason (e.g. requiring admin privileges for modifying certain reg hive keys etc).

Further in future while adding more config values, we would like to have such a flexibility.